chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v2
 name: hcloud-cloud-controller-manager
 type: application
-version: v1.15.0 # x-release-please-version
+version: v1.16.0 # x-release-please-version
```

Comment:
Please remove this change. The chart version will be updated automatically for new releases. You can check out #439 for an example of this.

Reply:
done